packages/vue/src/Form/MultipleUploadFile/OcMultipleUploadFile.vue

```diff
@@ -73,9 +73,7 @@ onMounted(() => {
       :label="label"
       :hint="hint"
       :error-message="
-        errorMessage
-          ? isErrorMaxSize && `File(s) is more than ${maxSize}MB`
-          : ''
+        isErrorMaxSize ? `File(s) is more than ${maxSize}MB` : errorMessage
```

Comment:
can we assume the default value for errorMessage is always `''`? 
asking because what happens when there is no error message. what will be displayed ?

Reply:
@Sammuel09 it will be display nothing